### ISSUE TITLE
report nan

### ISSUE THREAD

**Issue by JiaWang0704** [hsiangyuzhao/TBIOneShot#1]
Thank you very much for your work, why do I report nan every time I train to a certain epoch number, have the authors ever encountered such a problem, I am very much looking forward to your reply, it will help us to cite your paper!

**Comment by hsiangyuzhao** [comment]
Hi @JiaWang0704 Thank you for your interest in our work.

The nan value problem could encounter due to the domain gap between moving and fixed images. We have encountered this problem when transforming our method to CANDI dataset. But such issues did not happen when using on our sTBI dataset.

A carefully tuned learning rate could alleviate this problem. Also, you may tune the hyperparameters of the registration module. Another attempt is to pretrain the registration and then freeze it.

**Comment by JiaWang0704** [comment]
Thank you for your detailed response. Your suggestions are very valuable to our reproduction efforts.

**Comment by CherryNhu** [comment]
First of all, thank you so much for your contribution. 

My name is Cherry, and I am a Computer Science student at the University of Information Technology (UIT) - VNUHCM. I am currently conducting research in this field, and I found your paper incredibly exciting and insightful.

To further my understanding and to attempt to replicate and build upon your results, I would be very grateful if I could get access to the dataset used in your study.

I have the following questions regarding the dataset:
1.  Is the dataset publicly available? If so, could you please point me to the download link?
2.  If it is not publicly available, would it be possible for you to share it with me for academic and research purposes?
3.  Are there any specific terms of use, citation requirements, or data use agreements I need to follow?

I understand that the data is the result of significant effort, and I assure you that it will be used strictly for non-commercial, academic research. I would, of course, be happy to cite your paper and this repository appropriately in any of my future publications or reports.

Thank you for your time and for considering my request. I greatly appreciate your contribution to the research community.
